variables.tf

```diff
@@ -38,6 +38,16 @@ variable "deployment_name" {
   default     = "three-tier-app"
 }
 
+variable "database_type" {
+  type            = string
+  description     = "Cloud SQL Database flavor, mysql or postgres"
+  default         = "postgres"
```

Comment:
**Heads-up:** Reminder to communicate in [b/350774994](http://b/350774994) the value the Console dropdown should pass into Terraform (`postgres` — not `postgresql`). Either works for me.

Reply:
Chose to go with postgresql to be consistent with out public docs, and not requiring a UI update. thanks!